README.md

```diff
@@ -160,51 +160,57 @@ $ astro dev --help
 $ astro deploy --help
 ```
 
-## Development
+## Local Development
 
 How to get started as a developer:
 
-1. Build:
+1. Install `go` 1.7+ - https://go.dev/doc/install
+2. Install `golangci-lint` to run linter locally
 
-```
-$ git clone git@github.com:astronomer/astro-cli.git
-$ cd astro-cli
-$ make build
-```
+    ```brew install golangci-lint```
 
-1. (Optional) Install to `$GOBIN`:
+    ```golangci-lint run .```
+3. Install `pre-commit` to run lint on every commit
 
-```
-$ make install
-```
+    ```brew install pre-commit```
 
-1. Run:
+    ```pre-commit install```
 
-```
-$ astro
-```
+    Run lint locally:
 
-### Testing Locally
+    ```pre-commit run --all-files```
+4. Clone and Build:
 
-astro-cli is a single component of the much larger Astronomer Enterprise platform. In order to test locally you will need to
+```
+$ cd ~go/src/github.com/astronomer/astro-cli
```

Comment:
I don't think I changed that path

Reply:
for me the src is not there may be different for others

Reply:
That is the path I use

Reply:
It should have been `cd $GOPATH/src/github.com/astronomer` since locally folks can have different GOPATH and `astro-cli` shouldn't be in the path, as we clone and cd into `astro-cli` later

Reply:
I'll update